requirements.txt

```diff
@@ -0,0 +1,20 @@
+# Home Assistant Core
```

Comment:
More requirements files 😆

Reply:
Yes, I need something to hash, hashing the setup.py feels weird.